### DISCUSSION TITLE
reCaptcha Selenium Python

### DISCUSSION THREAD

**Discussion by Felipe-BT** [frontendbr/forum#1851] [Perguntas e Respostas]
Estou automatizando uma pagina com o Selenium, porém ao entrar no site e fazer o login manualmente ele trava na hora do Captcha, por mais que eu responda corretamente ele não funciona. Tem alguma maneira de `esconder` que o navegador está sendo usado por um robô?

**Comment by portothree** [comment]
Você pode iniciar o webdriver com um user agent diferent pro selenium, aqui há uma lista que pode usar:
https://developers.whatismybrowser.com/useragents/explore/

Entretanto, a depender da quantidade de requests que fizer, não vai fugir dos captchas.

```js
const webdriver = require('selenium-webdriver');
const FirefoxProfile = require('firefox-profile');

const profile = new FirefoxProfile().setPreference('general.useragent.override', 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/44.0.2403.157 Safari/537.36');

```

**Reply by canalqb** [reply]
`https://youtu.be/HQfTUkCcpcw`

**Reply by canalqb** [reply]
https://canalqb.blogspot.com/search?q=Selenium
Friends, I created several Selenium tips, I hope it helps someone, please, if you like it, like the blog, a hug.

**Comment by Felipe-BT** [comment]
Mudei manualmente e deu certo.
Para usar com o chrome é só trocar o 'Firefox' por Chrome ?

**Comment by portothree** [comment]
> Mudei manualmente e deu certo.
> Para usar com o chrome é só trocar o 'Firefox' por Chrome ?

É preciso preciso trocar o webdriver, no exemplo que dei seria pro gecko (firefox)
